datalake/src/main/java/com/sequenceiq/datalake/cm/ClouderaManagerProxiedClientFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.datalake.cm;
+
+import com.cloudera.api.swagger.client.ApiClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class ClouderaManagerProxiedClientFactory {
+
+    private static final String API_VERSION_32 = "v32";
+
+    private static final int CM_READ_TIMEOUT_MS =  60 * 1000;
+
+    private static final int CM_CONNECT_TIMEOUT_MS = 20 * 1000;
```

Comment:
why 20sec? I think we use 1 min in CB for this

Reply:
The cm module have these already configured to use proxy and direct connections, I think that should be used instead of creating new CM related code in other modules.

Reply:
I think we used 20 seconds in thunderhead repo. I've updated this to 60 sec. As mentioned in previous comment, I think it makes sense to move the logic into other modules as a separate ticket.